### PR TITLE
updated code to fix version issues in go libraries and k8s

### PR DESCRIPTION
**Pull request by vineet-krishna**

### CODE REVIEW

File: script/entrypoint.sh [hunk 1]
```diff
@@ -2,23 +2,9 @@
 set -ex
 
 # Render a AWS credentials with user and role for `awsudo`.
-DEFAULT_REGION=us-east-1
+DEFAULT_REGION=ap-southeast-1
```
Comment (yash-mindtickle) [inline script/entrypoint.sh]: Is default region required ?

File: README.md [hunk 2]
```diff
@@ -25,10 +25,8 @@ Example using standard AWS SDK credentials and an assumed role:
 #!/bin/bash
 set -ex
 docker run --rm -it \
-    -e ACCESS_KEY_ID=${ACCESS_KEY_ID:?}
-    -e SECRET_ACCESS_KEY=${SECRET_ACCESS_KEY:?}
-    -e ROLE_ARN=${ROLE_ARN:?}
     -e CLUSTER=your-cluster-name \
     -e AUTOSCALING_GROUPS=${AUTOSCALING_GROUP:?} \
+    -v "$HOME/.aws:/.aws"
```
Comment (yash-mindtickle) [inline README.md]: Can you push this to ecr, and then readme main local setup ye wala as it is chod de, and for everyone else, add a step like `./rotate-nodegroups.sh role1 role2` and this script will fetch asgs via roles and rotate those

File: script/entrypoint.sh [hunk 3]
```diff
@@ -2,23 +2,9 @@
...
 REGION=${REGION:-${DEFAULT_REGION}}
```
Comment (yash-mindtickle) [inline script/entrypoint.sh]: Also add capability to take different profiles, as of now, it will only work for the default aws profile right ?

Comment (vineet-krishna) [reply script/entrypoint.sh]: added

File: script/entrypoint.sh (hunk 1 above)
Comment (vineet-krishna) [reply script/entrypoint.sh]: option to override region is also present

File: kubectl-plugin/rotate_nodes [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      help="true"
+      shift # past argument=value
+    ;;
+    *)
+      # ignore
+      shift # past argument=value
+    ;;
+  esac
+done
+
```
Comment (yash-mindtickle) [inline kubectl-plugin/rotate_nodes]: Don't take cluster and region as inputs, use these
```
cluster=$(cat $KUBECONFIG | grep 'current-context' | awk '{print $2}' | cut -d/ -f 2)
region=$(cat $KUBECONFIG | grep 'current-context' | awk '{print $2}' | cut -d: -f 4)
```

File: build.sh [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+echo "Building version '$version'"
+set -ex
+docker build -t rotate-eks-asg:$version .
+if [[ $push == "true" ]]; then
+  docker tag rotate-eks-asg:$version 191195949309.dkr.ecr.ap-southeast-1.amazonaws.com/devops/rotate-eks-asg:$version
+  aws ecr get-login --no-include-email --region ap-southeast-1 | sh -
+  docker push 191195949309.dkr.ecr.ap-southeast-1.amazonaws.com/devops/rotate-eks-asg:$version
+fi
+set +ex
```
Comment (yash-mindtickle) [inline build.sh]:
```
docker_build:
	docker build -t 191195949309.dkr.ecr.ap-southeast-1.amazonaws.com/yash/pinger:latest .

docker_push:
	docker push 191195949309.dkr.ecr.ap-southeast-1.amazonaws.com/yash/pinger:latest
```

**Pull request merged**